README.md

```diff
@@ -1,10 +1,60 @@
-# Website
+# Project Layout
+
+### The project is broken down into 4 sections
+- Concepts - General Uniswap information or concepts useful for using Uniswap products, such as *Liquidity* and *Fees*
+- Contracts - The Uniswap smart contracts such as *Permit2*, or groupings such as the V3 Contracts
```

Comment:
super nit: reverse order here for familiarity as permit2 isn't as known as the core protocol contracts

```suggestion
- Contracts - Uniswap smart contracts such as the V3 Contracts or *Permit2*
```